basic_datatypes.ipynb

```diff
@@ -2105,6 +2310,15 @@
     "from typing import TypeVar, Hashable, Any\n",
     "\n",
     "def solution_dict_return_delete_value(my_dict: dict[Hashable, Any], key: Hashable) -> Any:\n",
+    "    \"\"\"Deletes and returns the value of a key in a dictionary, if it exists, otherwise returns None\n",
```

Comment:
I would write more explicitely "the value associated with a key in a dictionary if the key exists. Otherwise, return none".